scripts/run_profile_pyuvsim.py

```diff
@@ -63,18 +63,23 @@
 
     # Baseline selection:
     input_uv.baseline_array = np.repeat(input_uv.baseline_array[:args.Nbls], args.Ntimes)
-    input_uv.ant_1_array, input_uv.ant_2_array = input_uv.baseline_to_antnums(input_uv.baseline_array)
+    input_uv.ant_1_array, input_uv.ant_2_array = input_uv.baseline_to_antnums(
+        input_uv.baseline_array)
     ants_new = np.unique(input_uv.ant_1_array.tolist() + input_uv.ant_2_array.tolist())
     input_uv.antenna_numbers = ants_new
-    input_uv.antenna_names = ants_new.astype(str)  # Antnames/numbers are going to be messed up by the baseline selection. Unimportant.
+    input_uv.antenna_names = ants_new.astype(
+        str)  # Antnames/numbers are going to be messed up by the baseline selection. Unimportant.
```

Comment:
put comment on line above to avoid code line split

Reply:
Done